### PR TITLE
Add actions for API used by Root Coding robot

### PR DESCRIPTION
**Pull request by justinIRBT**

### CODE REVIEW

File: action/LedAnimation.action [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Request
+# Supported Animation types
+int8 BLINK_LIGHTS = 1
```
Comment (justinIRBT) [inline action/LedAnimation.action:3]: My thought is that people can just publish to the topic for Root's On/Off "animation" so I didn't include those here

**Comment by alsora** [comment]
Minor comment: the geometry_msgs messages (e.g. Pose and Quaternion) express values as double (`float64` in IDL).
I think we should use that in the input as well to be consistent.

**Comment by alsora** [comment]
I think we should be more expressive in the feedback and result field.
See the nav2 actions as example https://github.com/ros-planning/navigation2/tree/main/nav2_msgs/action
They often include things like "distance traveled" and "current pose" in the feedback, and "elapsed time" in the output.

**Comment by justinIRBT** [comment]
> I think we should be more expressive in the feedback and result field. See the nav2 actions as example https://github.com/ros-planning/navigation2/tree/main/nav2_msgs/action They often include things like "distance traveled" and "current pose" in the feedback, and "elapsed time" in the output.

Regarding the feedback, I am providing the same result as root https://github.com/iRobotEducation/root-robot-ble-protocol#command-17---navigate-to-position-finished-response

I can provide more intermediate feedback, like distance to goal

**Review by alsora — APPROVED** [review]

**Pull request merged**